tests/updateIds_test.js

```diff
@@ -561,4 +561,126 @@ describe('update ids', () => {
       expect(updatedFile).not.to.include('@T1d6a52b9');
     });
   });
+
+  describe('[Playwright examples] lines processing', () => {
+    it('[ts file]: the same import name as suite name', () => {
+      const analyzer = new Analyzer('playwright', 'virtual_dir');
+      analyzer.withTypeScript();
+      require('@babel/core');
+
+      const idMap = {
+        tests: {
+          'test case #1': '@T1d6a52b9',
+        },
+        suites: {
+          Example: '@Sf3d245a7',
+        },
+      };
+
+      mock({
+        node_modules: mock.load(path.resolve(__dirname, '../node_modules')),
+        virtual_dir: {
+          'test.ts': `
+          import { test, page } from '@playwright/test';
+          import Example from '@src/Example';
+
+          test.describe('Example', () => {          
+            test('test case #1', async ({ page }) => {
+
```

Comment:
```suggestion
const userId = 1
```
Please check that this const won't be updated or reordered